cli/src/main/java/com/box/l10n/mojito/cli/command/GitBlameCommand.java

```diff
@@ -0,0 +1,298 @@
+//            commandHelper.waitForPollableTask(pollableTask.getId());
+
+        } catch (PollableTaskException e) {
+            throw new CommandException(e.getMessage(), e.getCause());
+        }
+    }
+
+    /**
+     * Get the git-blame information for given line number
+     * @param lineNumber
+     * @param blameResultForFile
+     * @param gitBlameWithUsage
+     * @return
+     */
+    private void getBlameResults(int lineNumber, BlameResult blameResultForFile, GitBlameWithUsage gitBlameWithUsage) {
+        GitBlame gitBlame = new GitBlame();
+
+        gitBlameWithUsage.setGitBlame(gitBlame);
+
+        gitBlame.setAuthorName(blameResultForFile.getSourceAuthor(lineNumber).getName());
+        gitBlame.setAuthorEmail(blameResultForFile.getSourceAuthor(lineNumber).getEmailAddress());
+        gitBlame.setCommitName(blameResultForFile.getSourceCommit(lineNumber).toString());
+        gitBlame.setCommitTime(Integer.toString(blameResultForFile.getSourceCommit(lineNumber).getCommitTime()));
+    }
+
+    /**
+     * Builds a git repository if current directory is within a git repository
+     * @return
+     * @throws CommandException
+     */
+    org.eclipse.jgit.lib.Repository getGitRepository() throws CommandException {
+
+        if (gitRepository == null) {
+            FileRepositoryBuilder builder = new FileRepositoryBuilder();
+
+            try {
+                gitRepository = builder
+                        .findGitDir(new File(sourceDirectoryParam))
+                        .readEnvironment()
+                        .build();
+            } catch (IOException ioe) {
+                throw new CommandException("Can't build the git repository");
+            }
+        }
+
+        return gitRepository;
+    }
+
+    /**
+     * Get the git-blame information for entire file
+     * @param filePath
+     * @return
+     * @throws CommandException
+     */
+    BlameResult getBlameResultForFile(String filePath) throws CommandException {
+
+        try {
+            org.eclipse.jgit.lib.Repository gitRepository = getGitRepository();
+
+            BlameCommand blamer = new BlameCommand(gitRepository);
+            ObjectId commitID = gitRepository.resolve("HEAD");
+            blamer.setStartCommit(commitID);
+            blamer.setFilePath(filePath);
+            BlameResult blame = blamer.call();
+
+            return blame;
+        } catch (GitAPIException | IOException e) {
+            String msg = MessageFormat.format("Can't get blame result for file: {0}", filePath);
+            logger.error(msg, e);
+            throw new CommandException(msg, e);
+        }
+    }
+
+    /**
+     * Checks if the given line contains text unit(s)
+     * @param line
+     * @param gitBlameWithUsages
+     * @return list of GitBlameWithUsage objects that match current line
+     */
+    List<GitBlameWithUsage> getTextUnitNameFromLine(String line, List<GitBlameWithUsage> gitBlameWithUsages) {
```

Comment:
the function should be rename since not it return a list of GitBlameWithUsage for that line and not a text unit name anymore